pcdsdevices/interface.py

```diff
@@ -1020,6 +1119,8 @@ def _remove_methods(self):
                 delattr(obj, method_name)
             except AttributeError:
                 pass
+            if hasattr(obj, '_tab'):
+                obj._tab.remove(method_name)
```

Comment:
Rather than managing these individually, we probably could have done blanket regexes e.g. `wm_.*`